binder/apt.txt

```diff
@@ -0,0 +1,11 @@
+dbus-x11
+firefox
```

Comment:
Do we need really to install Firefox? I mean, how much overhead does it add to the deployment time @yuvipanda?

Reply:
This is cached, as long as dependencies do not change I think this does not add much overhead.

Reply:
Good catch. Was leftover from another demo. Removed